crates/bounded_planet_core/src/bin/client/main.rs

```diff
@@ -67,7 +67,11 @@ async fn run(options: Opt) -> Result<(), Box<dyn std::error::Error>> {
         accept_any_cert: options.accept_any_cert
     });
 
-    app.add_system(log_connections.system());
+    app.add_resource(PingResponderState::default());
```

Comment:
Not sure if you want to, but this is also equivalent:
```suggestion
    app.init_resource::<PingResponderState>();
```